README.md

```diff
@@ -1,113 +1,143 @@
-# <img src="https://cdn.rawgit.com/theupdateframework/artwork/3a649fa6/tuf-logo.svg" height="100" valign="middle" alt="TUF"/> A Framework for Securing Software Update Systems
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/shubhusion/python-tuf/blob/develop/tuf-horizontal-white.png">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/shubhusion/python-tuf/blob/develop/tuf-horizontal-black.png">
+  <img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="https://github.com/shubhusion/python-tuf/blob/develop/tuf-horizontal-black.png">
+</picture>
```

Comment:
* alt text is incorrect
* I don't think you need to specify `source` for light theme: the default `img` is enough
* I think linking to pngs in cncf/artwork project would be totally fine, even preferable: if you do want to upload the pngs to this project then
   * please don't put them directly in the root directory
   * `https://github.com/shubhusion/python-tuf/` is not ok as url